deploy/kubernetes/base/controller.yaml

```diff
@@ -19,9 +19,6 @@ spec:
         beta.kubernetes.io/os: linux
       serviceAccount: ebs-csi-controller-sa
       priorityClassName: system-cluster-critical
-      tolerations:
```

Comment:
Why changing controller deployment is needed too? I think you are only changing node service daemon set

Reply:
It seems odd to have a specific use case toleration for the controller, so I removed it too. Is there a community preference about controllers running on tainted nodes by default that should be followed here?

Reply:
I don't remember the exact reason why this was added. But I believe this is due to similar reason as [here](https://github.com/kubernetes/kubernetes/pull/46748). So that the driver could still be scheduled onto the node which is tainted with `CriticalAddonsOnly`

Reply:
I didn't realize that was upstream convention (thanks for sending that along). I've added toleration back to the controller.